RFS/src/main/java/org/opensearch/migrations/bulkload/transformers/TransformFunctions.java

```diff
@@ -75,17 +76,21 @@ public static void removeIntermediateMappingsLevels(ObjectNode root) {
              * - [{"doc":{"properties":{"address":{"type":"text"}}}}]
              * - [{"audit_message":{"properties":{"address":{"type":"text"}}}}]
              *
-             * It's not impossible that the intermediate key is not present, in which case we should just extract the mappings:
+             * It's also possible for this list to be empty, in which case we should set the mappings to an empty node.
```

Comment:
Why is it appropriate to replace [] with "[{}]".  Is this true for all versions of targets?

Reply:
This results in an index creation call that looks like:

```
curl -X PUT "localhost:92011/index_name" -H "Content-Type: application/json" -d '
{
  "settings": {
        <the settings>
  },
  "mappings": {
  }
}'
```

I'm not sure if the empty mappings blob is valid for ALL possible targets, but it does appear valid for the targets we currently consider - OS 1.X & OS 2.X